src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefFunctionalTest.java

```diff
@@ -265,9 +266,11 @@ public void shouldGetOnlyFewCaseWorkerDetails() {
         assertTrue(StringUtils.isNotEmpty(caseWorkerProfile.getFirstName()));
         assertTrue(StringUtils.isNotEmpty(caseWorkerProfile.getLastName()));
         assertTrue(StringUtils.isNotEmpty(caseWorkerProfile.getOfficialEmail()));
+        assertTrue(StringUtils.isNotEmpty(caseWorkerProfile.getStaffAdmin()));
```

Comment:
Hi @rajkatla-hmcts  
1) Value may return  true or false. while creating the user. if they create a as admin it will return true or other cases it is false. is it make sense ?